project/src/main/scala/edu/gemini/osgi/tools/idea/IdeaProject.scala

```diff
@@ -157,11 +157,7 @@ class IdeaProject(idea: Idea, scalaInstance: ScalaInstance, imls: List[File]) {
       </component>
       {rootComponent(javaVersion)}
       <component name="ProjectRunConfigurationManager"/>
-      <component name="ScalaCompilerConfiguration">
-        <plugins>
-          <plugin path="$PROJECT_DIR$/../../../lib/compile/paradise_2.10.5-2.0.1.jar" />
-        </plugins>
-      </component>
+      <component name="ScalaCompilerConfiguration"/>
```

Comment:
I am a little bit irritated that we are able to delete this so soon after I figured it out.